session1/examples/generics.ts

```diff
@@ -0,0 +1,33 @@
+//Generic type variables
+function checkPasswordLength<T>(password: T) {
+    console.log(password.length);
+}
+
+//Generic types
+function testArray<T>(item: Array<T>) {
+    return item;
+}
+let furits = testArray<string>(["Apples"]);
+furits.push(100);
+
+//Generic classes
+class stack<T>{
+    constructor(public item: Array<T>) {
```

Comment:
1. This should be a member variable of the class itself. 
2. Why are we overriding the default constructor with a blank constructor? Let's put some dummy logic.